gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -225,6 +228,10 @@ public TransportChannelProvider withEndpoint(String endpoint) {
     return toBuilder().setEndpoint(endpoint).build();
   }
 
+  public TransportChannelProvider setAllowHardBoundTokens(ArrayList<String> allowedValues) {
```

Comment:
Since this method is not on the interface anymore, I don't think we have to expose a separate setter for it. Exposing it on the builder should be good enough.